src/protocols/aeternity/libs/AeternityAdapter.ts

```diff
@@ -194,8 +198,15 @@ export class AeternityAdapter extends BaseProtocolAdapter {
     // TODO
   }
 
-  async fetchTransactions(): Promise<any> {
-    // TODO
+  async fetchTransactions(
+    address:string,
+    nextPageParams:string | null,
```

Comment:
I suggest to return `undefined` instead of `null` so it can be typed with the use of question mark:
```
nextPageParams?: string
```